secure_message/services/case_service.py

```diff
@@ -11,16 +11,21 @@ class CaseService:
     def store_case_event(case_id, user_name):
         """posts the data to the case service"""
 
-        json_data = {"description": "New Secure Message",
-                     "category": "SECURE_MESSAGE_SENT",
-                     "createdBy": user_name}
+        if case_id:
+            json_data = {"description": "New Secure Message",
+                         "category": "SECURE_MESSAGE_SENT",
+                         "createdBy": user_name}
```

Comment:
What happens if the user_name is somehow empty?  Should there be some checking for this if it's important?

Reply:
This is being handled where this function is called in the _inform_case_service which only calls this if it has a non empty name.

Reply:
Is it also checking for a not None `case_id`?